swap_meet/clothing.py

```diff
@@ -0,0 +1,7 @@
+from swap_meet.item import Item
+class Clothing(Item):
+    def __init__(self, category = None, condition = 0, year = 0):
```

Comment:
Since you don't use `category` in the body of your initializer you should remove it from the argument list.

```suggestion
    def __init__(self, condition = 0, year = 0):
```